src/components/styles/Dropdown.js

```diff
@@ -0,0 +1,93 @@
+import styled from "styled-components";
+
+export const DropdownContainer = styled.div`
+  position: relative;
+  display: block;
+  margin-bottom: 20px;
+`;
+
+export const DropdownLabel = styled.label`
+  display: block;
+  margin-bottom: 8px;
+  font-size: 14px;
+  color: #333;
+`;
+
+export const IconWrapper = styled.span`
+  display: inline-block;
+  display: flex;
```

Comment:
I'm not 100% sure, but I guess you can only have one option for ```display:```and not define it twice